### PR TITLE
Helm Chart Refactoring

### PR DESCRIPTION
**Pull request by palexster**
# Description

This PR refactors the code of the Liqo helm chart (Ref. #460):

- [x] Remove existing subcharts
- [x] Splits templates into different files to simplify troubleshooting

# How Has This Been Tested?

* [x] E2E Testing

### CODE REVIEW

File: deployments/liqo/templates/auth-service-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          - "--namespace"
+          - "$(POD_NAMESPACE)"
+          - "--resyncSeconds"
+          - "29"
+          - "--listeningPort"
+          {{ if .Values.authService.ingress.enable }}
+          - "4999"
+          {{ else }}
+          - "442"
+          - "--useTls"
```
Comment (aleoli) [inline deployments/liqo/templates/auth-service-deployment.yaml:46]: Is there any reason why these 3 numbers have been decreased by 1? (the same for the ingress and the service)

File: deployments/liqo/templates/discovery-deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      containers:
+        - image: {{ .Values.discoveryOperator.image.repository }}{{ .Values.global.suffix | default .Values.suffix }}:{{ .Values.global.version | default .Values.version }}
+          name: discovery
+          imagePullPolicy: {{ .Values.pullPolicy }}
+          command: ["/usr/bin/entrypoint.sh", "/usr/bin/discovery"]
+          args:
+          - "--namespace"
+          - "$(POD_NAMESPACE)"
+          - "--requeueAfter"
+          - "29"
```
Comment (aleoli) [inline deployments/liqo/templates/discovery-deploy.yaml:30]: same as before

File: deployments/liqo/templates/auth-service-deployment.yaml (hunk 1 above)
Comment (palexster) [reply deployments/liqo/templates/auth-service-deployment.yaml:46]: very strange

File: deployments/liqo/templates/auth-service-service.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    run: auth-service
+  ports:
+    - name: https
+      protocol: TCP
+      {{ if .Values.authService.ingress.enable }}
+      port: 4999
+      targetPort: 4999
+      {{ else }}
+      port: 442
+      targetPort: 442
```
Comment (giorio94) [inline deployments/liqo/templates/auth-service-service.yaml:25]: Shouldn't those be 5000 and 443 (at least those are the ports specified in the deployment). Also in the ingress the port is wrong

**Review by AbakusW — APPROVED** [review]

File: deployments/liqo/values.yaml [hunk 4]
```diff
@@ -120,3 +119,26 @@ global:
...
+    repository: "liqo/virtual-kubelet"
+    pullPolicy: "IfNotPresent"
+
+initVk:
+  image:
+    repository: "liqo/init-vkubelet"
+    pullPolicy: "IfNotPresent"
+
+suffix: ""
+version: "latest"
```
Comment (aleoli) [inline deployments/liqo/values.yaml:144]: These two fields are already set at the beginning of the file, and there are also inside the global object

```yaml
suffix: ""
version: "latest"
...
global:
  suffix: ""
  version: ""
...
suffix: ""
version: "latest
```

I think that we can delete some of them

**Review by aleoli — APPROVED** [review]

**Pull request merged**